Source/options.cpp

```diff
@@ -1419,6 +1418,255 @@ uint32_t KeymapperOptions::KeyForAction(string_view actionName) const
 	return SDLK_UNKNOWN;
 }
 
+PadmapperOptions::PadmapperOptions()
+    : OptionCategoryBase("Padmapping", N_("Padmapping"), N_("Padmapping Settings"))
+{
+	buttonToButtonName.emplace(ControllerButton_AXIS_TRIGGERLEFT, "LT");

```

Comment:
nit: Construct in one go, as in `: buttonToButtonName({{ControllerButton_AXIS_TRIGGERLEFT, "LT"}, ...)`